const geneticsRegulationBio =       {
   index: 13,
   itemName: "Gene Regulation",
   folderName: "Gene Regulation",
   type: "Molecular",
   entries: [
      {
        "index": 0,
        "title": "Gene Regulation",
        "type": "NORMAL",
        "image": null,
        "data": [
         {
            "index": 0,
            "name": null,
            "value": "●Transfer of genetic info from DNA to RNA to protein."
          },           
          {
            "index": 0,
            "name": null,
            "value": "●DNA is made up of DNA\n●DNA is used to give instru­ctions for the production of proteins in the process of protein synthesis\n●Gene regulation determines which genes are turned on/off\n●Proteins can increase or decrease transc­ription"
          }
        ]
      },
      {
        "index": 0,
        "title": "Codons",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "● mRNA triplet that codes for an amino acid\n● start codon: AUG\n● stop codon: UAA, UAG, UGA\nreading frame:\n● start to stop sequence of nitrogen bases\nantico­dons:\n● complement of the codon found on tRNA"
          }
        ]
      },
      {
        "index": 0,
        "title": "Differ­ent­iation",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●When a cell changes from one type to another\n●All specia­lized cells come from stem cells (unspe­cia­lized)\n●DNA contains genes and genes contain proteins that change the way cells look and act\n●Every somatic cell in your body contain the same DNA\n●Using genes → expressing → turned \"­on\"\n●The specia­lized cells can not specialize again and can not go backwards to the stem cells\n●Cells decide what they will be based on internal or external enviro­nmental cues\n●internal: transc­ription factors will activate certain genes and turn them on (factors are bunched up because of when the zygote will divide)\n●external: (induc­tion) (like peer pressure) a group of cells can induce another group to differ­entiate by using signals (like diffusion, direct contace, gap junctions)\n●goal: to change gene expression (turn on/off genes"
          }
        ]
      },
      {
        "index": 0,
        "title": "3 Types of RNA",
        "type": "TABLE_LIST",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "mRNA",
            "value": "synthe­­sized using DNA template, attaches to ribosome in cytoplasm and specifies the primary structure of protein"
          },
          {
            "index": 0,
            "name": "rRNA",
            "value": "molecu­­le­s...and proteins make up the ribos­omes"
          },
          {
            "index": 0,
            "name": "tRNA",
            "value": "translates between nucleic acid (DNA) and protein lang. by carrying specific amino acids to ribosome, where they recognize the approp­­riate codons in the mRNA"
          }
        ]
      },
      {
        "index": 0,
        "title": "Main Types of RNA",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "mRNA:\n● \"­mes­sen­ger­\"\n● carries genetic code to the ribosome\n● codon\ntRNA:\n● \"­tra­nsf­er\": transfers amino acids to the ribosome\n● anticodon\nrRNA:\n● \"­rib­oso­mal­\": makes up ribosomes\n● ribosomes build proteins"
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA",
        "type": "NORMAL",
        "image": "DNA",
        "data": []
      },
      {
        "index": 0,
        "title": "DNA Comparison",
        "type": "TABLE",
        "headers": [
          "Prokar­yotic DNA:",
          "Eukaryotic DNA:"
        ],
        "image": null,
        "data": [
          {
            "index": 0,
            "columns": [
              {
                "value": "double stranded"
              },
              {
                "value": "double stranded"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "circular"
              },
              {
                "value": "linear"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "one chromosome"
              },
              {
                "value": "usually more than one chromosome"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "in cytoplasm"
              },
              {
                "value": "in nucleus"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "no histones"
              },
              {
                "value": "DNA wrapped around histones (proteins)"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "superc­oiled DNA"
              },
              {
                "value": "forms chromatin"
              }
            ]
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA methyl­ation",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●\"off switch­\" tightly wrapped around histones\n●genes can not be transc­ribed\n●methyl groups are added to the DNA\n●gene expression is reduced\n●less transc­ription\n●barr bodies: one X chromosome condenses because of DNA methyl­ation"
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA Packing",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●chromatin: a complex of DNA and protein\n●histones: proteins associated with DNA packing"
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA Replication",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "\tIn S phase of Mitosis making DNA from DNA nucleo­tides can only be added to the 3' end of a nucleotide 5' to 3' direction enzymes mediate the process of DNA replic­ation\n1) helicase unwinds DNA at origin of replic­ation and creates replic­ation forks\n2) topois­omerase prevents overwi­nding and single­-strand binding proteins support the replic­ation bubble\n3) primase adds RNA primer\n4) DNA polymerase III adds nucleo­tides in 5' to 3' direction on leading strand\n5) lagging strand grows in 3' to 5' direction away from the replic­ation fork by the addition of okazaki fragments\n6) DNA ligase seals together okazaki fragments (short segments of DNA that grow 5' to 3' that are added onto the lagging strand)\n7) DNA polymerase replaces RNA primers with DNA"
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA Replication Image",
        "type": "NORMAL",
        "image": "DNA_REPLICATION_IMAGE",
        "data": []
      },
      {
        "index": 0,
        "title": "DNA Replication Key Factors",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●Eukary­otic: replic­ation before mitosis or meiosis (inter­phase)\n●helicase: unzipping enzyme\n● breaks the hydrogen bonds holding bases together\nDNA polyme­rase: builder\n● replicates DNA molecules to build new strand of DNA\nprimase: initia­lizer\n● makes the primer so that DNA polymerase can figure out where to go to start to work\nligase: gluer (binder)\n● helps glue/bind DNA fragments together"
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA Replic­ation Process (2nd example)",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●starts at the origin (ident­ified by DNA sequence)\n1) helicase unwinds DNA\n- single stranded binding protein bind to DNA strands to prevent the strands from going back together\n- topois­omerase keeps DNA from superc­oiling\n2) primase makes RNA primers on both strands\n3) DNA polymerase builds new strand in 5' to 3' direction\n- this means it moves along old template strand in 3' to 5' direction\n- adds new bases to 3' end on new strand\n4) ligase takes care of gaps between Okazaki fragments\nat the end of replic­ation there is two identical DNA molecules\n- semi-c­ons­erv­ative: each copy contain a new and original strand"
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA Replic­ation Steps Image",
        "type": "NORMAL",
        "image": "DNA_REPLICATION",
        "data": []
      },
      {
        "index": 0,
        "title": "DNA Structure & Replication",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "Stru­cture of DNA",
            "value": "* Each DNA nucleotide is made up of 5-carbon sugar called deoxyr­ibose, a phosphate group, and a nitrog­enous base.\n* DNA uses bases A, C, G, & T. (RNA uses A, C, G, & U)"
          },
          {
            "index": 0,
            "name": "Double Helix",
            "value": "* DNA has an antipa­rallel structure→ The 2 strands run in opposite directions of eachother.\n* Each strand has a 5' end and a 3' end."
          },
          {
            "index": 0,
            "name": "DNA Replic­ation",
            "value": "* DNA is Semi-­Con­ser­vative\n→Each of the 2 strands in DNA acts as a template to produce 2 new strands.\n* Enzymes \"­unz­ip\" DNA molecules by breaking the hydrogen bonds that hold the two strands together.\n* Primary enzyme involved is DNA polymerase\n→ Joins nucleo­tides to synthesize the new comple­mentary strand.\n→Proof­reads each DNA strand to prevent errors."
          },
          {
            "index": 0,
            "name": "Leading & Lagging Strand",
            "value": "* Leading Strand\n→runs 5' to 3' towards the fork and is made contin­uously.\n* Lagging Strand\n→runs 5' to 3' away from the fork and is made in small pieces called Okazaki fragments."
          },
          {
            "index": 0,
            "name": "Other Things to Know:",
            "value": "\n\tDNA polymerase only synthe­sizes DNA in the 5’ to 3’ direction only. The difference between the leading and lagging strands is that the leading strand is formed towards replic­ation fork, while the lagging strand is formed away from replic­ation fork.\n\tDNA replic­ation is not the same as cell division. Replic­ation occurs before cell division, during the S phase of the cell cycle. However, replic­ation only concerns the production of new DNA strands, not of new cells."
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA vs. RNA",
        "type": "TABLE",
        "headers": [
          "DNA",
          "RNA"
        ],
        "data": [
          {
            "index": 0,
            "columns": [
              {
                "value": "double stranded"
              },
              {
                "value": "single stranded"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "deoxyribose"
              },
              {
                "value": "ribose"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "A-T"
              },
              {
                "value": "A-U"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "G-C"
              },
              {
                "value": "G-C"
              }
            ]
          }
        ]
      },
      {
        "index": 0,
        "title": "DNA vs. RNA",
        "type": "TABLE",
        "headers": [
          "DNA",
          "RNA"
        ],
        "image": null,
        "data": [
          {
            "index": 0,
            "columns": [
              {
                "value": "Double Stranded, Anti-p­arallel"
              },
              {
                "value": "Single Stranded"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "A+T and C+G"
              },
              {
                "value": "A+U and C+G"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "Mostly Found in Nucleus"
              },
              {
                "value": "Mostly Found in Cytoplasm"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "Deoxyribose"
              },
              {
                "value": "Deoxyribose"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "Long Polymer"
              },
              {
                "value": "Much Shorter"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "Forms Double Helix Structure"
              },
              {
                "value": "Forms Secondary or Tertiary Structure"
              }
            ]
          }
        ]
      },
      {
        "index": 0,
        "title": "Eukaryotic pre-mRNA splicing",
        "type": "TABLE",
        "image": null,
        "data": [
          {
            "index": 0,
            "columns": [
              {
                "value": "Intr­ons"
              },
              {
                "value": "non-coding sequences"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "Exons"
              },
              {
                "value": "sequences that will be translated"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "Small ribonu­cle­opr­otein particles (snRNPs \"­snu­rps­\")"
              },
              {
                "value": "Looks for introns and exons and recognizes it."
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "Spli­ceo­somes"
              },
              {
                "value": "respon­sible for removing introns"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "snRNPs cluster with other proteins to form spli­ceo­some"
              }
            ]
          }
        ]
      },
      {
        "index": 0,
        "title": "Eukaryotic Transc­ription",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "3 different RNA polyme­rase:",
            "value": "\nRNA polymerase I: Transc­ribes rRNA.\nRNA polymerase II: treans­cribes mRNA and some snRNA.\nRNA polymerase III: transc­ribes tRNA and some other small RNAs\n●Each RNA polymerase recognizes it own promoter."
          },
          {
            "index": 0,
            "name": "Init­iat­ion of transc­rip­tion:",
            "value": "Requires a series of tran­scr­ption factors (helper).- Tran­scr­iption factors: Necessary to get the RNA polymerase II enzyme to a prom­oter* and to initiate gene expres­sion.\nElon­gat­ion: RNA transc­ribed from the DNA template.\nTerm­ina­tion not as well defined.\nInit­iat­ion of trancr­iption\n- Transc­ription factors bind to a promoter region and recruit RNA polyme­rase.\n- Forms the initation complex."
          },
          {
            "index": 0,
            "name": "Tran­scr­iption factors:",
            "value": "Necessary to get the RNA polymerase II enzyme to a prom­oter* and to initiate gene expres­sion."
          },
          {
            "index": 0,
            "name": "Elon­gat­ion:",
            "value": "RNA transc­ribed from the DNA template.\nTerm­ina­tion not as well defined."
          },
          {
            "index": 0,
            "name": "Init­iat­ion of trancr­iption:",
            "value": "\n- Transc­ription factors bind to a promoter region and recruit RNA polyme­rase.\n- Forms the initation complex."
          }                   
        ]
      },
      {
        "index": 0,
        "title": "Evolution of the Codes",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "\t●Early evolution since shared among living species\n●Genes can be transf­erred within species and among others as well"
          }
        ]
      },
      {
        "index": 0,
        "title": "Frameshift mutations",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "- Addition or deletion of a single base\n- Much more profound conseq­uences\n- Alter reading frame downstream\n- Triplet repeat expansion mutation\n- Hunting disease: Repeat unit is expanded in the disease allele relative to the normal"
          }
        ]
      },
      {
        "index": 0,
        "title": "Histone Acetylation",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●\"on switch­\"\n●loosely wrapped around histones\n●genes can be transc­ribed\n●acetul groups are added to amino acids of histone proteins"
          }
        ]
      },
      {
        "index": 0,
        "title": "Lagging Strand",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●Primer is several nucleo­tides\nDNA primase goes along the lagging strand and adds RNA primer. Once you have primer, polymerase can add on DNA at 3' end (5' to 3')\n●End up with Okazaki fragments\n●Slower process\nDNA ligase puts all fragments together as one strand\n● RNA is replaced with DNA"
          }
        ]
      },
      {
        "index": 0,
        "title": "Leading Strand",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●Need RNA primer from DNA primase\n●RNA primer allows DNA polymerase to add nucleo­tides at the 3' end\n●Can not add nucleo­tides at 5' end"
          }
        ]
      },
      {
        "index": 0,
        "title": "The Nature of Genes",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "\tEarly ideas to explain how genes work came from studying human diseases.\n●Archibald Garrod proposed that patients with the disease alkap­ton­uria lacked a particular enzyme.\n●Beadle and Tatum studied Neurospora crassa. They looked for fungal cells lacking specific enzymes. Beadle and Tatum results was that each mutated enzyme disrupted one key enzyme in the metabolic pathway."
          }
        ]
      },
      {
        "index": 0,
        "title": "Terms",
        "type": "TABLE_LIST",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "DNA",
            "value": "Nucleic acid that transmits genetic inform­ation from parent to offspring and codes for the production of proteins"
          },
          {
            "index": 0,
            "name": "Nucleotide",
            "value": "Building block of nucleic acids"
          },
          {
            "index": 0,
            "name": "Double Helix",
            "value": "Structure of two strands, intert­wining around an axis like a twisted ladder"
          },
          {
            "index": 0,
            "name": "DNA replic­ation",
            "value": "Process during which a double­-st­randed DNA molecule is copied to produce two identical DNA molecules"
          },
          {
            "index": 0,
            "name": "Base",
            "value": "Principle in which the nitrog­enous bases of the DNA molecules bond with one another (AT, CG))"
          }
        ]
      },
      {
        "index": 0,
        "title": "Redundancy and Ambiguity of the Code",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●Redund­ancy: more than one codon for an amino acid\n●Ambiguity: codon do not code for more than one amino acid"
          }
        ]
      },
      {
        "index": 0,
        "title": "Regulation of Gene Expression",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "What makes cells different:",
            "value": "\n● cells have different shapes and proteins\n● cells use the DNA in the nucleus differ­ently\n● some gene are turned on/off"
          },
          {
            "index": 0,
            "name": "Differ­ent­ation:",
            "value": "\n● when a cell changes from one form to another\n● cells become specia­lized in structure and function"
          },
          {
            "index": 0,
            "name": "Differ­ential gene expres­sion:",
            "value": "\n● the expression of different genes by cells with the same genome"
          }                    
        ]
      },
      {
        "index": 0,
        "title": "Prokaryotic Transcription",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "Prok­aryotic Transc­rip­tion: Single RNA polymerase\n- Initiation of mRNA synthesis does not require a primer\nProk­aryotic Transc­ription requires a Promoter, Start Site, and a termin­ation site.\nTransc­ription occurs in three major stages:\n- Initiation\n- Elongation\n- Termin­ation\nInit­iat­ion: RNA polymerase binds to the prom­oter\nProm­oter: Forms a recogn­ition and binding site for the RNA polyme­rase.\n- Found upstream of the start site.\n- Not transc­ribed.\n- Asyme­tri­cal: indicate site of initiation and direction of termin­ation.\nElon­gat­ion: RNA transcript grows in the 5'-to-3' direction as ribonu­cle­otides are added.\n- Tran­scr­iption bubble: contains RNA polyme­rase, DNA template, and growing RNA transc­ript.\n- After the transc­ription bubble passes, the now-tr­ans­cribed DNA is rewound as it leaves the bubble.\nTerm­ina­tion: Marked by sequence that signals \"­s­top­\" to polyme­rase.\n- Causes the formation of phosph­odi­ester bonds to cease.\n- RNA-DNA hybrid within the transc­ription bubble dissoc­iates.\n- RNA polymerase releases the DNA.\n- DNA rewinds.\nHair­pin in RNA causes RNA polymerase to pause\nU:A base pairs weaken the DNA/RNA bonding.\nProkar­yotic tran­scr­ipt­ion is coupled to tran­sla­tion\n- mRNA begins to be translated before transc­ription is finished."
          }
        ]
      },
      {
        "index": 0,
        "title": "Prokaryotic vs. Eukaryotic Transciption",
        "type": "TABLE",
        "headers": [
          "Porkaryotic",
          "Eukaryotic"
        ],
        "image": null,
        "data": [
          {
            "index": 0,
            "columns": [
              {
                "value": "takes place in cytoplasm"
              },
              {
                "value": "takes place in nucleus"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "several gene transc­ribed at one time"
              },
              {
                "value": "single gene transc­ribed at one time"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "no modifi­cations before transl­ation"
              },
              {
                "value": "primary transcript modified before transl­ation"
              }
            ]
          }
        ]
      },
      {
        "index": 0,
        "title": "Prokar­yotic vs. Eukaryotic Transl­ation",
        "type": "TABLE",
        "headers": [
          "Prokaryotic",
          "Eukaryotic"
        ],
        "image": null,
        "data": [
          {
            "index": 0,
            "columns": [
              {
                "value": "takes place in cytoplasm"
              },
              {
                "value": "takes place in cytoplasm"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "ribosomes begin transl­ating while mRNA is still transc­ribing"
              },
              {
                "value": "transc­ription and transl­ation separate"
              }
            ]
          }
        ]
      },
      {
        "index": 0,
        "title": "Parts of a Nucleotide",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "●phosphate group\n●sugar\n●nitrog­enous base"
          }
        ]
      },
      {
        "index": 0,
        "title": "Protein Targeting",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "In eukary­otes, transl­ation may occur in the cytoplasm or the rough endopl­asmic reticulum (RER).\nSignal sequen­ces at the beginning of the polype­ptide sequence bind to the signal recogn­ition (SRP).\n- The signal sequence and SRP are recognized by RER receptor proteins\n- Docking holds ribosome to RER\n- Beginning of the protei­n-t­raf­ficking pathway"
          }
        ]
      },
      {
        "index": 0,
        "title": "Riboomes",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "made in nucleotide\nP site: holds the polype­ptide\nA site: holds amino acids\nE site: exit site\nsome are free and some are fixed"
          }
        ]
      },
      {
        "index": 0,
        "title": "tRNA and Ribosomes",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "tRNA moleules carry amino acids to the ribosome for incorp­oration into a polype­ptide.\n- Amin­oac­yl-tRNA synthe­tase add amino acids to the acceptor stem of tRNA.\n- Anti­codon loop contains 3 nucleo­tides comple­mentary to mRNA codons."
          }
        ]
      },
      {
        "index": 0,
        "title": "tRNA charging reaction",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "Each aminoa­cyl­-tRNA synthetase recognizes only 1 amino acid but several tRNAs.\nCharged tRNA has an amino acid added using the energy from ATP.\n-Can undergo peptide bond formation without additional energy.\nRibosomes do not verify amino acid attached to tRNA.\nThe ribo­some has multiple tRNA binding sites:\nP site: binds the tRNA attached to the growing pe­ptide chain\nA site: binds the tRNA carrying the next amino acid.\nE site: binds the tRNA that carried the last amino acid, tRNA exits ribosome.\nThe ribo­some has two primary functi­ons\n- Decode the mRNA.\n- Form peptide bonds.\nPeptidyl transf­era­se:\n- Enzymatic component of the ribosome.\n- Forms peptide bonds between amino acids."
          }
        ]
      },
      {
        "index": 0,
        "title": "RNA Processing",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "Eukaryotic modifi­cations to primary transcript (pre mRNA)\n● before it leaves the nucleus\n● bond altera­tions to the ends\n● removal in interv­ening sequences"
          }
        ]
      },
      {
        "index": 0,
        "title": "mRNA modifi­cations",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "In euka­ryo­tes the primary transcript must be modified to become mature mRNA\nAddition of a 5' cap Protects nucleo­tides from getting lost, from degrad­ation.\n- Involved in transl­ation initia­tion.\nAddition of a 3' poly-A tail Created by poly-A polyme­rase, protection from degrad­ation\n- Puts whole string of A's (AAA) to protect!\nRemoval of noncoding sequences (int­ron­s): Pre-mRNA spli­cing done by splice­osome.\n- Cut it out to get rid of it!!!"
          }
        ]
      },
      {
        "index": 0,
        "title": "Operon",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "operon: way of regulating genes and is usually made up of a few genes that involve enzymes\nRNA polyme­rase: builder enzyme, needed in order to start transc­rip­tion, needs a promoter to bind to DNA\n                  operator: a part of the DNA where a repressor can bind, if repressor is bound to operator it blocks RNA polymerase which means mRNA can not be made so neither can proteins\nlac operon: operator and promoter region of DNA and three genes that code for enzymes that help in breaking down lactose\n● there is a gene that codes for the repressor production and this gene has its own promoter\n● if lactose is not present, then the repressor binds to the operator and blocks RNA polymerase which means mRNA and proteins can not be produced\n● if lactose is present, the lactose (sugar) binds to the repressor (repressor can not bind to operator) and RNA polymerase finds its promoter, binds, and transc­ribes to make mRNA from the genes on operon, the mRNA will be used to make enzymes to break down the lactose sugar\n● no lactose: \"­off­\"\nrp operon:\n● evolved in bacteria to deal with absence of tryptophan\n● tryptophan is on amino acid which moves proteins\n● designed to make tryptophan if it is not present\n● if bacteria does not have trypto­phan, there is a number of genes that are required to make it\n● tryptophan fits inside the repressor and the repressor will change it's shape to fit in the receptor\n● if a lot of tryptophan is present, then we do not want to make more so the repressor is going to set operator in \"­off­\""
          }
        ]
      },
      {
        "index": 0,
        "title": "The Lac Operon",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "The Lac Operon of E.Coli",
            "value": "* Condit­ions: Lactose is available and Glucose is not."
          }
        ]
      },
      {
        "index": 0,
        "title": "Role of Introns",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "regulate gene activity\nsingle gene may be able to synthesize more than one protein"
          }
        ]
      },
      {
        "index": 0,
        "title": "Structure of DNA",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "double helix\n● \"­bac­kbo­ne\": sugar + phosphate● \"­run­gs\": nitrog­enous bases"
          }
        ]
      },
      {
        "index": 0,
        "title": "Ther Genetic Code",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "Francis Crick and Sydney Brenner determined how the order of nucleo­tides in DNA encoded amino acid order.\n- They introduced single nucleotide insertions or deletions and looked for mutations (Fra­meshift mutati­ons)\nA Codon is a block of three DNA nucleo­tides corres­ponding to an amino acid,\nSpaced Codons: Codon sequence in a gene punctu­ated.\nUnspaced Codons: codons adjacent to each other.\n- Marshall Nirenberg identified the codons that specify each amino acid.\nStop Codon: 3 codons (UAA, UGA, UAG) used to terminate transl­ation\nStart Codon: Codon (AUG) used to signify the start of transl­ation\nCode is dege­ner­ate: Some amino acids are specified by more that one codon.\nCode practi­cally unicer­sal: Strongest evidence that all living things share common ancest­ry.\n- Advanced in genetic engine­ering.\n- Mitoch­ondria and cloroplast have some differ­ences in \"­sto­p\" signals."
          }
        ]
      },
      {
        "index": 0,
        "title": "Transcript Modifications",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "5' cap: GTP is added\ntwo functions:\n1) protects transcript from hydrolytic enzymes\n2) tags the end as \"­leader segmen­t\" for the ribosome\ns' end: last to be translated\npoly(A­)tail: 30 to 200 nucleo­tides added to end\n● inhibits degrad­ation\n● facili­tates ribosomal attachment\n● attached to stop codon\nRNA splicing\n● removal of introns (noncoding sequences) (inter­vening sequences)\n● pasting of exons (coded sequences) (exit the nucleus)\nsmall nuclear ribonu­cle­opr­oteins found in nucleus (snRNP; snurps): complexes of small RNA units and proteins found in nucleus\nsplice­osome: complex of snurps involved in the locating and cutting out of introns"
          }
        ]
      },
      {
        "index": 0,
        "title": "Transcription(DNA → RNA)",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "1) Initiation",
            "value": "\n● promoter sites: region of the DNA where the RNA polymerase binds\n● 100 nucleo­tides long\n● transc­ription factors: binding protein, TATA box: promoter sequence"
          },
          {
            "index": 0,
            "name": "2) Elongation",
            "value": "\n● RNA polymerase in action\n● separates and untwists helix\n● links nucleo­tides in a 5' to 3' direction\n3) Termin­ation\n● termin­ation sequence: AAUAAA"
          },
          {
            "index": 0,
            "name": "3) Termin­ation",
            "value": "\n● termin­ation sequence: AAUAAA"
          }          
        ]
      },
      {
        "index": 0,
        "title": "Transcription",
        "type": "NORMAL",
        "image": "REPLICATION",
        "data": [
          {
            "index": 0,
            "name": "Tran­scr­iption Key Points",
            "value": "* Involves copying a gene's DNA sequence to make an RNA molecule.\n* Performed by RNA polymerase\n* 3 Stages: Initia­tion, Elonga­tion, Termin­ation.\n* RNA molecules are spliced and have a 5' cap and poly-A tail put on their ends. (Eukar­yotes) }"
          },
          {
            "index": 0,
            "name": "Init­iation, Elonga­tion, Termin­ation",
            "value": "* RNA polymerase binds to a sequence of DNA called the promoter, found near the beginning of a gene. Each gene (or group of co-tra­nsc­ribed genes, in bacteria) has its own promoter. Once bound, RNA polymerase separates the DNA strands, providing the single­-st­randed template needed for transc­rip­tion.\n* One strand of DNA, the template strand, acts as a template for RNA polyme­rase. As it \"­rea­ds\" this template one base at a time, the polymerase builds an RNA molecule out of comple­mentary nucleo­tides, making a chain that grows from 5' to 3'. The RNA transcript carries the same inform­ation as the non-te­mplate (coding) strand of DNA, but it contains the base uracil (U) instead of thymine (T).\n* Sequences called termin­ators signal that the RNA transcript is complete. Once they are transc­ribed, they cause the transcript to be released from the RNA polyme­rase. An example of a termin­ation mechanism involving formation of a hairpin in the RNA is shown below."
          }
        ]
      },
      {
        "index": 0,
        "title": "Transcription and Translation Image",
        "type": "NORMAL",
        "image": "TRANSCRIPTION_TRANSLATION",
        "data": []
      },
      {
        "index": 0,
        "title": "Transcription Image",
        "type": "NORMAL",
        "image": "TRANSCRIPTION",
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "phosphate group\nsugar\nnitrog­enous base"
          }
        ]
      },
      {
        "index": 0,
        "title": "Transc­ription & Transl­ation",
        "type": "NORMAL",
        "image": "TRANSCRIPTION_and_TRANSLATION",
        "data": []
      },
      {
        "index": 0,
        "title": "Transc­ription and Transl­ation",
        "type": "TABLE_LIST",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "The Central Dogma",
            "value": "Inform­ation only flows from: DNA→­RNA­→­protein\nFirst described by Francis Crick."
          },
          {
            "index": 0,
            "name": "Tran­scr­ipt­ion",
            "value": "DNA→ RNA\n- DNA-di­rected synthesis of RNA\n- Only template strand of DNA used\n- T in DNA replaced by U in RNA.\n- mRNA used to direct synthesis of polype­ptides."
          },
          {
            "index": 0,
            "name": "Tran­sla­tion",
            "value": "- Synthesis of polype­ptides.\n- Takes place at ribosome.\n- Requires several kinds of RNA."
          },
          {
            "index": 0,
            "name": "RNA",
            "value": "All synthe­sized from DNA template by transc­ription\n- Messenger RNA (mRNA).\n- Ribosomal RNA (rRNA).\n- Transfer RNA (tRNA).\n- Small nuclear RNA (snRNA)\n- Signal recogn­ition particle RNA (SRP RNA).\n- Micro-RNA (miRNA)."
          }
        ]
      },
      {
        "index": 0,
        "title": "Translation",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "RNA → protein\ninform­ation in RNA is passed to proteins\n1) codon recogn­ition\n2) peptide bond formation\n3) transl­ocation"
          }
        ]
      },
      {
        "index": 0,
        "title": "Translation",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "Process by which the mRNA transcript is read by the ribosomes and used to make a polype­ptide.\nOccurs in 3 main stages:\n- Initiation\n- Elongation\n- Termin­ation"
          },
          {
            "index": 0,
            "name": null,
            "value": "There are some important differ­ences between transl­ation in prokar­yotes and eukary­otes."
          },
          {
            "index": 0,
            "name": null,
            "value": "- Ribosome binding sequence (RBS) of mRNA positions small subunit correctly.\n- Large subunit now added.\n- Initiator tRNA bound to P site with A site empty."
          },
          {
            "index": 0,
            "name": null,
            "value": "Init­iat­ions in eukar­yotes similar except:\n- Initiating amino acid is meth­ion­ine.\n- Lack of an RBS – small subunit binds to 5′ cap of mRNA."
          },
          {
            "index": 0,
            "name": null,
            "value": "Elon­gat­ion adds amino acids\n- 2nd charged tRNA can bind to empty A site\n- Requires elongation factor called EF-Tu to bind to tRNA and GTP\n- Peptide bond can then form.\n- Addition of successive amino acids occurs as a cycle."
          },
          {
            "index": 0,
            "name": null,
            "value": "- There are fewer tRNAs than codons\n- Wobble pairing allows less stringent pairing between the 3′ base of the codon and the 5′ base of the anticodon\n- This allows fewer tRNAs to accomm­odate all codons"
          },
          {
            "index": 0,
            "name": null,
            "value": "Term­ina­tion\n- Elongation continues until the ribosome encounters a stop codon\n- Stop codons are recognized by release factors which release the polype­ptide from the ribosome"
          }
        ]
      },
      {
        "index": 0,
        "title": "Translation",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "Translation",
            "value": "* tRNAs are molecular \"­bri­dge­s\" that connect mRNA codons to the amino acids they encode.\n* One end has an anticodon, which can bind to specific mRNA codons. (sequence of 3 nucleo­tides)\n* The other end carries the amino acid specified by the codons."
          },
          {
            "index": 0,
            "name": "Init­iation, Elonga­tion, Termin­ation",
            "value": "* The ribosome assembles around the mRNA to be read and the first tRNA (carrying the amino acid MET[AUG]). This initiation complex is needed in order for transl­ation to get started.\n* The mRNA is read one codon at a time, and the amino acid matching each codon is added to a growing protein chain.\n* Each time a new codon is exposed,\n→a matching tRNA binds to the codon\n→the existing amino acid chain (polyp­eptide) is linked onto the amino acid of the tRNA via a chemical reaction,\n→the mRNA is shifted one codon over in the ribosome, exposing a new codon for reading.\n* tRNAs move through the A, P, and E sites of the ribosome. This process repeats many times as new codons are read and new amino acids are added to the chain.\n* The finished polype­ptide chain is released. It begins when a stop codon (UAG, UAA, or UGA) enters the ribosome, triggering a series of events that separate the chain from its tRNA and allow it to drift out of the ribosome.\n* The polype­ptide may still need to fold into the right 3D shape, undergo proces­sin­g,get shipped to the right place in the cell, or combine with other polype­ptides before it can do its job as a functional protein."
          }
        ]
      },
      {
        "index": 0,
        "title": "Translation Image",
        "type": "NORMAL",
        "image": "TRANSLATION",
        "data": []
      },
      {
        "index": 4,
        "title": "Water/­Lip­id-­Soluble Hormone Signaling",
        "type": "NORMAL",
        "image": "WATER_LIPID_SOLUBLE_HORMONE_SIGNALING",
        "data": []
      },
      {
        "index": 0,
        "title": "Mutation: Altered Genes",
        "type": "TABLE_LIST",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "Point mutations",
            "value": "alter a single base"
          },
          {
            "index": 0,
            "name": "Base substi­tut­ion",
            "value": "substitute one base for another"
          },
          {
            "index": 0,
            "name": "Siletn mutation",
            "value": "same amino acid inserted"
          },
          {
            "index": 0,
            "name": "Missense mutation",
            "value": "changes amino acid inserted\n- Transi­tions\n- Transv­ersions"
          },
          {
            "index": 0,
            "name": "Nonsense mutati­ons",
            "value": "changed to stop codon\n"
          }
        ]
      },
      {
        "index": 0,
        "title": "Mutations",
        "type": "TABLE",
        "image": null,
        "data": [
          {
            "index": 0,
            "columns": [
              {
                "value": "point mutation",
                "styles": [
                  "BOLD"
                ]
              },
              {
                "value": "affects 1 nucleotide pair"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "1. silent mutations"
              },
              {
                "value": "do not change amino acid transl­ation"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "2.missense mutations"
              },
              {
                "value": "a single nucleotide change results in a codon that codes for a different amino acid"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "3. nonsense mutation"
              },
              {
                "value": "a regular amino acid codon is changed into a stop codon, ending transl­­ation"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "inse­rtion or deletion",
                "styles": [
                  "BOLD"
                ]
              },
              {
                "value": "additi­on/loss of nucleotide pairs"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "1. frame shift mutation"
              },
              {
                "value": "deletion or insertion in a DNA sequence that shifts the way the sequence is read"
              }
            ]
          },
          {
            "index": 0,
            "columns": [
              {
                "value": "mutagens",
                "styles": [
                  "BOLD"
                ]
              },
              {
                "value": "forces that interact with DNA in ways that cause mutation example: xrays"
              }
            ]
          }
        ]
      },
      {
        "index": 0,
        "title": "Mutations Image",
        "type": "NORMAL",
        "image": "MUTATIONS",
        "data": []
      },
      {
        "index": 0,
        "title": "Types of Mutations",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": "Point mutations:",
            "value": "\n● caused by just one nucleotide base pair substi­tution of a gene\n●Examples:\n\tmissense mutation:\n\t\t- still codes, but not properly (sickle cell anemia)\n\t nonsense mutation:\n\t\t altera­tions codes for a stop codon\n\t silent mutation:\n\t\t a change in DNA but not a change in the amino acid sequence"
          },
          {
            "index": 0,
            "name": "Frameshift mutations:",
            "value": "\n● caused by insertions and deletions of base pairs\n● alters the three letter reading frame"
          }          
        ]
      },
      {
        "index": 0,
        "title": "Chromosome Mutations",
        "type": "NORMAL",
        "image": null,
        "data": [
          {
            "index": 0,
            "name": null,
            "value": "● Changes the structure or number of chromo­somes"
          },
          {
            "index": 0,
            "name": "Deletion:",
            "value": "Loss of all or part of a chromosome."
          },
          {
            "index": 0,
            "name": "Duplic­ation:",
            "value": "Part of chromosome is copied."
          },
          {
            "index": 0,
            "name": "Inversion:",
            "value": "Reverses the direction of parts of a chromosome."
          },
          {
            "index": 0,
            "name": "Transl­oca­tion:",
            "value": "Part of one chromosome break off and attaches to another chromosome."
          }          
        ]
      }
    ]
};


export default geneticsRegulationBio;